launch.py

```diff
@@ -107,8 +106,8 @@ def git_clone(url, dir, name, commithash=None):
 if not is_installed("torch") or not is_installed("torchvision"):
     run(f'"{python}" -m {torch_command}', "Installing torch and torchvision", "Couldn't install torch")
 
-if not skip_torch_cuda_test:
-    run_python("import torch; assert torch.cuda.is_available(), 'Torch is not able to use GPU; add --skip-torch-cuda-test to COMMANDLINE_ARGS variable to disable this check'")
+if not skip_torch_cuda_test and not use_cpu:
+    run_python("import torch; assert torch.cuda.is_available(), 'Torch is not able to use GPU; add --skip-torch-cuda-test to COMMANDINE_ARGS variable to disable this check'")

```

Comment:
i think this change of `COMMANDLINE_ARGS` --> `COMMANDINE_ARGS` is an error?

Reply:
Yes, it is. Probably I accidentally deleted the "L", sorry.

Reply:
I just commited the correction of the typo (and removed the unused `import shlex` I forgot)